Content.Server/Salvage/SalvageSystem.ExpeditionConsole.cs

```diff
@@ -25,6 +26,21 @@ private void OnSalvageClaimMessage(EntityUid uid, SalvageExpeditionConsoleCompon
     {
         var station = _station.GetOwningStation(uid);
 
+        // Frontier
+        var activeExpeditionCount = 0;
+        var expeditionQuery = AllEntityQuery<SalvageExpeditionDataComponent, MetaDataComponent>();
+        while (expeditionQuery.MoveNext(out var expeditionUid, out _, out _))
+            if (TryComp<SalvageExpeditionDataComponent>(expeditionUid, out var expeditionData) && expeditionData.Claimed)
+                activeExpeditionCount++;
+
+        if (activeExpeditionCount >= 15)
```

Comment:
if this is a global limit on # of concurrent expeditions, it should be broken out into a CVAR for servers to set

Reply:
Sure ill rework it